docs/docs/how-to/using-yarn.md

```diff
@@ -0,0 +1,72 @@
+# Using Yarn
+
+## What is Yarn?
+
+[Yarn](https://yarnpkg.com/) is a package manager for JavaScript. It is used to manage and install dependencies for JavaScript projects, particularly for Node.js applications. Yarn provides a faster and more reliable alternative to the npm package manager, offering features like parallel package installations and offline caching. Yarn uses a `yarn.lock` file to control and reproduce consistent installations of dependencies across different environments.
+
+## Installing yarn
+
+> "The preferred way to manage Yarn is through [Corepack](https://nodejs.org/dist/latest/docs/api/corepack.html), a new binary shipped with all Node.js releases starting from 16.10." <br />-[from the Yarn documentation](https://yarnpkg.com/getting-started/install)
+
+Corepack is included with all Node.js >=16.10 installs, but you must opt-in. To enable it, run the following command:
+
+```bash
+corepack enable
+```
+
+## Using the correct version of yarn
+
+To see the version of yarn that you have installed, run the following command:
+
+```bash
+yarn --version
+```
+
+**Redwood requires Yarn (>=1.15)**
+
+You can upgrade yarn by running the following command:
+
+```bash
+corepack prepare yarn@stable --activate
+```
+
+:::info
+If this command fails, you may need to [uninstall the current version of Yarn first](/docs/how-to/using-yarn#uninstalling-yarn).
```

Comment:
Clicking on this link sent me to the main version of the docs; we want the version to be preserved, and since this heading is in the same page we can just use the `#...`

```suggestion
If this command fails, you may need to [uninstall the current version of Yarn first](#uninstalling-yarn).
```